src/Storages/StorageFuzzJSON.cpp

```diff
@@ -481,7 +481,11 @@ class FuzzJSONSource : public ISource
     {
         Columns columns;
         columns.reserve(block_header.columns());
-        columns.emplace_back(createColumn());
+        for (const auto& col : block_header)
```

Comment:
```suggestion
        for (const auto & col : block_header)
```
(https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/development/style.md)